src/components/Cards/Card.js

```diff
@@ -0,0 +1,59 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Card = ({
+  children,
+  className,
+  accented,
+  aggregated,
+  aggregatedMini,
+  equalized,
+  cardRef,
+  ...props
+}) => {
+  const equalizedCSS = {
```

Comment:
again I don't know if we want to do this... I think PF Core can create this CSS eventually. I think my hope short term was that we'd be able to use Equalizer the same way we do currently with jquery-match-height. We will need feedback from @jgiardino and @jeff-phillips-18 though.

Reply:
I would prefer to leave this out and let the application handle resizing cards to be the same height in a row. This should mean the card will take up the full height of its parent container. The application can deal with setting the appropriate parent heights. Leaving access to the `ref` is fine.

Reply:
I don't think it is going to be that trivial.

Without applying flexbox behavior, you won't be able to set height for a card without damaging the card layout.

I made this sandbox to show this case:
https://codesandbox.io/s/jp25kx754y

Considering this behavior,
**shouldn't cards suppose to be a flexbox by default from patternfly-core?**

@priley86 in can't find it, but I remember @gilad215 share your equalizer code with me, and you basically handled it by equalizing the card-body itself.
It works, but if in one row you have one card with a footer and one without a footer, it breaks.

Reply:
I'm OK with the styling to use flexbox until we get the correct styling from PF core. I would much rather see it in less/sass and not as inline styling. Does it need to be an option or should it always use flexbox?

Reply:
Yes, I believe it should always use flexbox.
It will allow consumers to easily set a height for a card.

@gilad215 can you move the styles into the less/scss as described here:
https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md#adding-styling-for-your-components